.github/workflows/deploy.yaml

```diff
@@ -20,7 +20,7 @@ on:
           - ctc.demo.getyourrefund.org
           - ctc.staging.getyourrefund.org
           - datadog
-          - demo.efilerapi.org
+          - demo.efiler-api.fileyourstatetaxes.org
```

Comment:
The `plan.yaml` workflow should also reflect this change.

Reply:
huh, we never had it in there in the first place. oops!